lib/twilioconnection.js

```diff
@@ -247,6 +299,41 @@ class TwilioConnection extends StateMachine {
     this.emit('error', new Error(reason));
   }
 
+  /**
+   * Handle an incoming "busy" message.
+   * @param {{cookie: string, keepAlive: boolean, retryAfter: number}} message
+   * @private
+   */
+  _handleBusy({ cookie, keepAlive, retryAfter }) {
+    if (this.state !== 'connecting') {
+      return;
+    }
+    const reason = retryAfter < 0
+      ? 'Received terminal "busy" message'
+      : `Received "busy" message, retrying after ${retryAfter} ms`;
+
+    if (retryAfter < 0) {
+      this._handleBad({ reason });
+      return;
+    }
+    // TODO(mmalavalli): Remove check for the existence of "cookie" once VSG supports it.
```

Comment:
I added a comment to the doc, and @ykhodosh agreed that the cookie should remain optional in busy.